shared/gh/js/views/tenant-admin/gh.admin-manage-orgunits.js

```diff
@@ -0,0 +1,260 @@
+/*!
+ * Copyright 2015 Digital Services, University of Cambridge Licensed
+ * under the Educational Community License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+define(['gh.core', 'gh.constants', 'jquery.jeditable'], function(gh, constants) {
+
+    /**
+     * Add an organisational unit
+     *
+     * @private
+     */
+    var addOrgUnit = function() {
+        // Get the form data
+        var displayName = $('#gh-manage-orgunits-modal-text').val();
+        var type = $('#gh-manage-orgunits-modal-type').val();
+        var parentId = parseInt($('#gh-manage-orgunits-add-modal').attr('data-id'), 10) || null;
+        var groupId  = parseInt($('#gh-manage-orgunits-add-modal').attr('data-groupid'), 10) || null;
+
+        // Create the organisational unit
+        gh.api.orgunitAPI.createOrgUnit(gh.data.me.AppId, displayName, type, parentId, groupId, null, null, false, function(err, data) {
```

Comment:
All this to say that you can leave groupId `null` (but please do verify this with a few users, it's been a while)